arangod/Agency/Node.cpp

```diff
@@ -22,15 +22,15 @@ const Node::Children Node::dummyChildren = Node::Children();
 const Node Node::_dummyNode = Node("dumm-di-dumm");
 
 static std::string const SLASH("/");
-static std::regex const reg("/+");
-
+static std::regex const duplicateSlashesRegex("/+");
```

Comment:
I thought we had a hand written function for that already. (without regex)

Reply:
True. I found that in arangod/Agency/Store.cpp we had an alternative `split(...)` function for agency keys.
That function was a lot better than the one in arangod/Agency/Node.cpp, so I added a test and verified that the two functions produced the same results, and finally removed the one in Node.cpp.
Now we only have split and normalize functions in Store.cpp, and they are optimized.
std::regex is not used anymore in Node.cpp or Store.cpp